features/storage/TESTS/kvstore/static_tests/main.cpp

```diff
@@ -31,13 +31,20 @@ static char         buffer[20] = {};
 static const size_t data_size = 5;
 static size_t       actual_size = 0;
 static const size_t buffer_size = 20;
+
```

Comment:
Having target specific settings inside generic test cases isn't something we can allow. What kind of issue you're trying to solve here?

Reply:
NUMAKER_M252KG just has 32KiB SRAM and meet OOM error in the `kvstore-static_tests` test. So I decrease the number of forked threads from 3 to 2 to reduce memory footprint. I will send another PR to address this issue separately.